src/components/common/CardSection.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View } from 'react-native';
+
+const CardSection = props => (
```

Comment:
do we have plans to use this component beyond where it's used currently? if not, it seems a little too unnecessary of an abstraction